docker-compose-db.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# The project Docker compose file for development.
+
+version: '2.3'
+
+services:
+  config: &db-config
+    environment:
+      - PGHOST=db
+      - PGHOST_SLAVE=db
+      - PGSSLMODE=prefer
+  geoportal: *db-config
+  # geoportal-advance: *db-config
+  tools: *db-config
+  alembic: *db-config
+  # alembic-advance: *db-config
+  # webpack_dev_server: *db-config
+
+  db:
```

Comment:
For the local docker db, I'm used to create an `env.local` file and I put the `db` service in `docker-compose.override.yaml`
With this I do not need to add `-f docker-compose.yaml -f docker-compose-db.yaml` on command line.

This is a little more complicated, but we need the local db in `make acceptance` (without `docker-compose.override.yaml`).

Note that in case of demo, it could be acceptable to have a local db in docker-compose.yaml, but this might cause conflicts on upgrades.

So let's go with this.

Reply:
What I wanted to do is to be able to:
- run the application on remote db
- debug the application on remote db
- test the application in the CI (required to have a local DB)

Then I think that's the simple way to do that...